src/components/Company/CompanyName/CorrectCompanyName.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-form ref="correctNameForm" v-model="valid" lazy-validation>
+    <v-layout row>
+      <v-flex>
+        <v-text-field
+          v-model="companyName"
+          filled
+          req
+          persistent-hint
+          :rules="companyNameRules"
+          data-test="business-identifier"
+        ></v-text-field>
+      </v-flex>
+    </v-layout>
+  </v-form>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Watch, Emit, Vue } from 'vue-property-decorator'
+import { Action, Getter, State } from 'vuex-class'
+
+// Interfaces && enums
+import { ActionBindingIF, BusinessInformationIF, NameRequestIF } from '@/interfaces'
+import { CorrectionTypes } from '@/enums'
+
+@Component({})
+export default class CorrectNameRequest extends Vue {
+  /** Form Submission Prop */
+  @Prop({ default: false }) submitId: string
```

Comment:
Type mismatch.

Reply:
Fixed in next commit!

Reply:
Refactored `submitId` to `formType` in all places in next commit.

Reply:
Should be
```
formType: CorrectionTypes
```

Reply:
gah you are right. Will amend this. 🤦

Reply:
Fixed

Reply:
It's still defaulted to '' ?

Reply:
Hmm, do i need a default? It would feel weird to default to a specific type?

Reply:
Null?

Reply:
Sure!

Reply:
Gaah, is the comment still correct for this prop?